jekyll/_data/sidenav.yml

```diff
@@ -458,6 +456,8 @@ en:
           link: https://circleci.com/docs/api/v2
         - name: API v1 reference
           link: https://circleci.com/docs/api/v1
+        - name: Configuration reference
```

Comment:
Should we consider moving this up before API v2 reference? I guess mostly because there's already an earlier API section and the config reference could be a bit more visible within this particular section..?